web/themes/new_weather_theme/templates/partials/alert-link.html.twig

```diff
@@ -11,7 +11,12 @@
     </svg>
   </div>
   <a
-    class="margin-top-05"
+    {#
+      Table cells have a set line height. Set our own line height so that we
+      know what we're working with. The margin nudges the text down a bit so it
+      vertically aligns with the icon.
+    #}
+    class="line-height-sans-1 margin-top-05"
```

Comment:
My suggestion is to change this to `line-height-sans-2` for longer/wrapping text. At token 1 it gets a little tight for those cases

Token 1:
![Screenshot of text with line-height-sans-1](https://github.com/weather-gov/weather.gov/assets/150970973/3b4d5f88-a522-4f11-98d1-ab81dc39dbd2)

Token 2:
![image](https://github.com/weather-gov/weather.gov/assets/150970973/0455101f-3b3d-46ba-975d-9ef95a794d80)